pyproject.toml

```diff
@@ -31,6 +31,7 @@ select = [
 ]
 ignore = [
     "COM812",   # -- over aggressively insists on trailing commas where not desireable --
+    "PT005",    # -- flags mock fixtures with names intentionally matching private method name --
     "PT011",    # -- pytest.raises({exc}) too broad, use match param or more specific exception --
```

Comment:
Does this change need to be replicated in in `.pre-commit-config.yaml` and `Makefile`? (note at the top)

Reply:
Hmm, yeah, good call. I've removed those redundant explicit options definitions. Now they should be picked up from the `pyproject.toml` so it's the single source of truth on options.